src/CLU/Microsoft.ScenarioTests.CLU/out.txt

```diff
@@ -0,0 +1,33 @@
+SETLOCAL EnableExtensions
+
+ECHO Test return code success case
+call testclu success show 2> out.error
+REM We expect that the commands above succeed (return 0)
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO "Expected error level 0 for success, got " %ERRORLEVEL%
```

Comment:
Normally, I’d agree. However, these tests are verifying that the CLU behaves correctly if there is an error. Thus, the info written to stderr by the cmdlet is not of interest to us and would only pollute the log.

From: Yugang Wang [mailto:notifications@github.com]
Sent: Thursday, December 17, 2015 4:45 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Johan Stenberg <Johan.Stenberg@microsoft.com>
Subject: Re: [azure-powershell] Update tests + escape paramters passed to clurun by (bash) wrapper script (#1519)


In src/CLU/Microsoft.ScenarioTests.CLU/out.txt<https://github.com/Azure/azure-powershell/pull/1519#discussion_r47982357>:

> @@ -0,0 +1,33 @@

> +SETLOCAL EnableExtensions

> +

> +ECHO Test return code success case

> +call testclu success show 2> out.error

> +REM We expect that the commands above succeed (return 0)

> +IF %ERRORLEVEL% NEQ 0 (

> +     ECHO "Expected error level 0 for success, got " %ERRORLEVEL%

That is fine. To me it would be normal to swallow the stdout but still let stderr surface to the console, but it is your call.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1519/files#r47982357>.